library/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db   import models
+
+class Library(models.Model):
+    user       = models.ForeignKey("user.User", on_delete = models.CASCADE)
+    book       = models.ForeignKey("book.Book", on_delete = models.CASCADE)
+    created_at = models.DateTimeField(auto_now_add = True)
+    read       = models.BooleanField()
```

Comment:
```suggestion
    read       = models.BooleanField(default=False)
```
nullable 한게 아니라면, default=False와 같이 기본값을 지정하는게 좋을것 같습니다. 
null 허용한다면 null=True를 써주세요.  
기존에는 NullBooleanField가 존재했으나 3.1에 Deprecated되었습니다.

Reply:
반영하였습니다.